mandatory/2-logic-error.js

```diff
@@ -1,16 +1,15 @@
 // The syntax for these functions is valid but there are some errors, find them and fix them
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
```

Comment:
Comment: Looks good and same answers as me on all three.

Reply:
Good work Saliha. 👍